src/components/CodeBlock.jsx

```diff
@@ -3,7 +3,8 @@ export default function CodeBlock({ exportedCodeString }) {
     <article className="cmp-code-block">
       <h2 className="cmp-code-block__heading">Code Example</h2>
       <section className="cmp-code-block__content">
-        <pre data-testid="pre-tag-found">
+        {/* eslint-disable-next-line jsx-a11y/no-noninteractive-tabindex */}
+        <pre data-testid="pre-tag-found" tabIndex={0}>
```

Comment:
When I ran a scan of the page with AxeDevTools, I got an error about the code block not being keyboard accessible. Because it scrolls, and initially there was no focus state on it, I don't think there was any sort of indicator to a keyboard-only user that there was more content being hidden. The suggestion AxeDevTools gave me was to add a `tabIndex` (and I'm pretty sure the resource/article I looked at was also suggested). Once I did that, then I could use the keyboard and tab right to the code block, I could use the arrow keys to scroll back and forth. It was pretty cool!

Reply:
Interesting! I didn't know scrolling sections needed to be tab-accessible, but that totally makes sense.

Reply:
So these code blocks may potentially have hidden content and include a scrollbar? Which is why you included it?